src/QuadForm/QuadBin.jl

```diff
@@ -442,24 +393,25 @@ function isequivalent(f::QuadBin{fmpz}, g::QuadBin{fmpz}; proper::Bool = true)
   if isindefinite(f)
     fred = reduction(f)
     gred = reduction(g)
-    if issquare(d)
```

Comment:
I guess this part makes no sense? :-D

Reply:
yes `_isequivalent_reducible` has taken care of this.

Reply:
so that part of  code is never executed

Reply:
So, should we remove it?

Reply:
o.k. after some further tests I am convinced that the `_isequivalent_reducible` works.